maintenance/CheckLastWikiActivity.php

```diff
@@ -19,6 +19,10 @@ public function execute(): void {
 	}
 
 	public function getTimestamp(): int {
+		$defaultActor = $this->getServiceContainer()->getUserFactory()
```

Comment:
Probably should have a test case in CheckLastWikiActivityTest to make sure properly excludes MediaWiki default edits.